crates/nu-utils/src/default_files/sample_config.nu

```diff
@@ -912,18 +914,18 @@ const NU_PLUGIN_DIRS = $NU_PLUGIN_DIRS ++ [($nu.default-config-dir | path join '
 
 # Appending to the OS path is a common configuration task.
 # Because of the previous ENV_CONVERSIONS (performed internally
-# before your config.nu loads), the path variable is a list that can 
+# before your config.nu loads), the path variable is a list that can
 # be appended to using, for example:
-$env.path ++= "~/.local/bin"
+$env.PATH ++ [ "~/.local/bin" ]
```

Comment:
```suggestion
$env.PATH ++= [ "~/.local/bin" ]
```

Right?

Reply:
These worked when I wrote them.  Then the `++` and `++=` operators were changed ;-).

Reply:
Huh, it used to raise a syntax error but no longer